### PR TITLE
add electron-ipc-stream

### PR DESCRIPTION
**Pull request by jprichardson**
https://github.com/jprichardson/electron-ipc-stream

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Some good apps written with Electron.
 - [cookies](https://github.com/hstove/electron-cookies) - Adds support for `document.cookie`.
 - [window](https://github.com/jprichardson/electron-window) - Easily create windows, pass arguments to renderer, and manage global window references.
 - [is-electron-renderer](https://github.com/jprichardson/is-electron-renderer) - Check if your code is running in `main` or `renderer`.
+- [ipc-stream](https://github.com/jprichardson/electron-ipc-stream) - Readable/Writable (Duplex) streams over IPC.
```
Comment (sindresorhus) [inline readme.md]: > Duplex stream over IPC.

Comment (jprichardson) [inline readme.md]: Is this what you want me to change it to?

Comment (sindresorhus) [inline readme.md]: Yeah

**Comment by jprichardson** [comment]
Fixed description.

**Pull request merged**